src/widget.ts

```diff
@@ -824,6 +825,39 @@ export class DSVEditor extends Widget {
     this._column = hit.column;
   }
 
+  /**
+   * A handler for the on mouse up signal
+   */
+  private _onMouseHover(
+    emitter: RichMouseHandler,
+    hoverRegion: 'ghost-row' | 'ghost-column' | null
+  ): void {
+    // Switch both to non-hovered state.
+    const style = { ...this._grid.extraStyle } as PaintedGrid.ExtraStyle;
+    if (this.grid.style.voidColor === '#F3F3F3') {
+      style.ghostColumnColor = 'rgba(243, 243, 243, 0.80)';
+      style.ghostRowColor = 'rgba(243, 243, 243, 0.80)';
+    } else {
+      style.ghostColumnColor = 'rgba(0, 0, 0, 0.65)';
+      style.ghostRowColor = 'rgba(0, 0, 0, 0.65)';
+    }
+    switch (hoverRegion) {
+      case null: {
+        break;
+      }
+      case 'ghost-row': {
+        style.ghostRowColor = 'rgba(0, 0, 0, 0)';
+        break;
+      }
+      case 'ghost-column': {
+        style.ghostColumnColor = 'rgba(0, 0, 0, 0)';
```

Comment:
Can we use the values in `this._grid.extraStyle` as much as possible, so that the styling stays in one place? Just so that when someone changes the styles, they only have to be changed in one place

Reply:
I'm exporting `LIGHT_EXTRA_STYLE` and `DARK_EXTRA_STYLE` and using those now so we only need to update in one place.